modules/dreamview/frontend/src/renderer/ground.js

```diff
@@ -12,6 +13,8 @@ export default class Ground {
         this.mesh = null;
         this.geometry = null;
         this.initialized = false;
+        this.inNavigationMode = null;
+
```

Comment:
inNaviMode instead of inNavigationMode??

Reply:
done